test/tests/outputs/checkpoint/autosavetimer.i

```diff
@@ -0,0 +1,55 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  ny = 10
+  uniform_refine = 2
+[]
+
+[Variables]
+  [./u]
+  [../]
+[]
+
+[Kernels]
+  [./diff]
+    type = CoefDiffusion
+    variable = u
+    coef = 0.1
+  [../]
+  [./time]
+    type = TimeDerivative
+    variable = u
+  [../]
+[]
+
+[BCs]
+  [./left]
+    type = DirichletBC
+    variable = u
+    boundary = left
+    value = 0
+  [../]
+  [./right]
+    type = DirichletBC
+    variable = u
+    boundary = right
+    value = 1
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+  num_steps = 50
+  dt = 0.01
+  solve_type = PJFNK
+  petsc_options_iname = '-pc_type -pc_hypre_type'
+  petsc_options_value = 'hypre boomeramg'
+[]
+
+[Outputs]
+  [cp]
+    type = TestAutosaveTimedCheckpoint
```

Comment:
This is the type of syntax I want to see, but it should just be `Checkpoint` right? We can and should add a whole sub-block like this for testing, overriding the autosave time - I don't have any issues with that. I don't believe we need a separate test object to verify that the default works if the override value works though.